WebApi/Program.cs

```diff
@@ -13,27 +14,12 @@ public class Program
     {
         public static async Task Main(string[] args)
         {
-            var host = CreateHostBuilder(args).Build();
-            using var scope = host.Services.CreateScope();
-            var services = scope.ServiceProvider;
-
-            try
-            {
-                var context = services.GetRequiredService<ItemDbContext>();
-                await context.Database.MigrateAsync();
-                await ItemSeed.SeedData(context);
-            }
-            catch (Exception exception)
-            {
-                var logger = services.GetRequiredService<ILogger<Program>>();
-                logger.LogError(exception, "An error occured during migration");
-            }
-           
-            await host.RunAsync();
```

Comment:
why was this removed?

Reply:
Because we do our migrations manually eitherway